.github/workflows/release.yml

```diff
@@ -86,10 +71,14 @@ jobs:
           # need this token in order to have it trigger the comment workflow
           GITHUB_TOKEN: ${{ secrets.NIGHTLY_PAT }}
 
-  stable:
-    if: github.repository == 'remix-run/remix' && github.event.action == 'published' && github.event_name != 'schedule'
+  manual:
```

Comment:
From what it looks like, I think we could take the nightly release job and put it into its own workflow file right?